Firestore/core/src/model/value_util.cc

```diff
@@ -36,6 +36,23 @@
 namespace firebase {
 namespace firestore {
 namespace model {
+namespace {
+
+/** The smallest reference value. */
+pb_bytes_array_s* kMinimumReferenceValue =
+    nanopb::MakeBytesArray("projects//databases//documents/");
+
+/** The field type of a maximum proto value. */
+std::string kRawMaxValueFieldKey = "__type__";
```

Comment:
FYI The linter wants this to be a char pointer.

Reply:
Done.